src/rich-text-container.component.js

```diff
@@ -75,6 +75,7 @@ const defaultContextValue = {
   addSerializer: noop,
   removeSerializer: noop,
   serialize: noop,
+  sanitizeHTML: noop,
```

Comment:
I think `noop` might cause bugs in this case. Because we want it to take in a string and return a string. Instead, maybe this?

```js
{
  sanitizeHTML(html) {
    return html
  }
}
```

Reply:
I'm pretty new to noop. How might a bug be encountered?

Reply:
I actually did this for a reason, but you are right that it won't return an htmlString if not set (which would be unexpected behavior).  My thought process here was that since the default sanitizeHTML function is defined on the RichTextEditor, and since that is also where the user can specify the sanitizeHTML function through a prop, this noop would sort of ensure that _IF_ the editor hadn't (for whatever reason) set the sanitizer up, we wouldn't return any html.  Sort of a fail safe to make sure that the correct sanitizer function had been set before we start returning html to eliminate the issue of returning html to a user that they thought was sanitized with their sanitizing function, but which wasn't (which would also be unexpected behavior... but worse in my opinion than not returning anything at all).  Does that make sense?

Reply:
If that's the case, we need to clarify the documentation in https://bandicoot.js.org/components/rich-text-editor.html where it says "optional but highly recommended"?

Also, this would be a breaking change to bandicoot -- requiring a new prop breaks existing usage. Which I'm okay with, but we'd just want to publish it as a new major version.

Reply:
Sorry if I wasn't clear before but this shouldn't be a breaking change and should not require a change to the "optional but highly recommended" text in the docs.  A default `sanitizeHTML` function **does** get set which merely returns the html (without sanitization)... see code lines #10 and #236 of rich-text-editor.component.js.  I'm was just saying that the RichTextEditor sets it up... not the RichTextContainer itself.  So the reason for the initial `noop` on the RichTextContainer is just to buy time for the RichTextEditor to do its thing.  Since the RichTextEditor itself is where the prop is that allows the user to specify the sanitizeHTML function, we need to make sure that we give the chance for the user to set theirs up before we do something that just returns unsanitized HTML.  However, if they don't provide a sanitizeHTML, the default is to return just the unsanitized HTML.  Does that make more sense or clarify anything?

Reply:
i looked at those lines and i think i get it now.

Reply:
ok